package.json

```diff
@@ -24,11 +24,12 @@
     "loader-utils": "^3.2.1",
     "mdb-react-ui-kit": "^8.0.0",
     "nextjs-websocket": "^1.0.11",
+    "pkg.json": "^2.0.8",
```

Comment:
@Nikhilesh002 what is the need for this?

Reply:
I dont know, it was suggested in terminal, then I installed it.